policies/base.rb

```diff
@@ -0,0 +1,3 @@
+default_source :supermarket
```

Comment:
Please rename base.rb to Policyfile_base because rake will create lock file for it.